src/views/SystemIntake/ContractDetails/index.tsx

```diff
@@ -165,57 +165,76 @@ const ContractDetails = ({
-                                value={source}
+                                name="Select an option"
+                                value=""
+                                disabled
                               />
-                            ))}
-                          </Field>
-                        </FieldGroup>
-                        <FieldGroup
-                          scrollElement="fundingSource.fundingNumber"
-                          error={!!flatErrors['fundingSource.fundingNumber']}
-                        >
-                          <Label htmlFor="IntakeForm-FundingNumber">
-                            Funding Number
-                          </Label>
-                          <FieldErrorMsg>
-                            {flatErrors['fundingSource.fundingNumber']}
-                          </FieldErrorMsg>
-                          <Field
-                            as={TextField}
+                              {fundingSources.map(source => (
+                                <Field
+                                  as={DropdownItem}
+                                  key={source.split(' ').join('-')}
+                                  name={source}
+                                  value={source}
+                                />
+                              ))}
+                            </Field>
+                          </FieldGroup>
+                          <FieldGroup
+                            scrollElement="fundingSource.fundingNumber"
                             error={!!flatErrors['fundingSource.fundingNumber']}
-                            id="IntakeForm-FundingNumber"
-                            maxLength={6}
-                            name="fundingSource.fundingNumber"
-                          />
-                        </FieldGroup>
+                          >
+                            <Label htmlFor="IntakeForm-FundingNumber">
+                              Funding Number
+                            </Label>
+                            <FieldErrorMsg>
+                              {flatErrors['fundingSource.fundingNumber']}
+                            </FieldErrorMsg>
+                            <Field
+                              as={TextField}
+                              error={
+                                !!flatErrors['fundingSource.fundingNumber']
+                              }
+                              id="IntakeForm-FundingNumber"
+                              maxLength={6}
+                              name="fundingSource.fundingNumber"
+                            />
+                          </FieldGroup>
+                        </div>
+                        <HelpText
+                          id="IntakeForm-FundingNumberHelp"
+                          className="margin-y-1 width"
+                        >
+                          <Link
+                            aria-label="Open 'Cookies opt-out' in a new tab"
```

Comment:
This `aria-label` looks wrong. I'm guessing it's leftover copy/paste.

Reply:
It definitely is a copy paste, good catch.